src/popup/modules/UserInterface.js

```diff
@@ -154,8 +154,10 @@ function selectAllText(event) {
         // but set scroll position to top one, because you want to see the
```

Comment:
Could you also adjust the comment then or at least move it to the correct position? Thanks.

Reply:
yeah, will do.